airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4450,6 +4512,25 @@ void loop(void) {
 		fetchSensorPPD(result_PPD);
 	}
 
+
+	if (cfg::rtc_read) {
+		DateTime now = rtc.now();
+		debug_outln_info("The RTC time is: ");
+		Serial.print(now.year(), DEC);
+		Serial.print('/');
+		Serial.print(now.month(), DEC);
+		Serial.print('/');
+		Serial.print(now.day(), DEC);
+		Serial.print(' ');
+		Serial.print(now.hour(), DEC);
+		Serial.print(':');
+		Serial.print(now.minute(), DEC);
+		Serial.print(':');
+		Serial.print(now.second(), DEC);
+		Serial.println();
+		fetchSensorRTC(result_RTC);
```

Comment:
@Warukira looking at the function result_RTC does not seem to be updated anywhere.

Reply:
I've resolved this.